docs: add a landing page to the documentation (#277)
widgetti/ipyvuetify: docs/index.rst

:html_theme.sidebar_secondary.remove:

.. raw:: html

    <!-- CSS overwrite for the landing page only -->
    <style>
    /* Make homepage wider */
    .bd-main .bd-content .bd-article-container {
        max-width: 80%;
    }
    /* hide the main title */
    h1 {display: none;}
    h2 {
        font-weight: bold;
        font-size: var(--pst-font-size-h1);
    }
    </style>

ipyvuetify: Jupyter widgets based on Vuetify UI components
==========================================================

.. raw:: html

    <div id="hero">
        <div id="hero-left">

IpyVuetify
----------

Jupyter widgets based on Vuetify UI components
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> **ipyvuetify** is a widget library for making modern looking GUI’s in Jupyter notebooks (`classic <https://github.com/jupyter/notebook>`__, `lab <https://github.com/jupyterlab/jupyterlab>`__, `lite <https://github.com/jupyterlite/jupyterlite>`__) and dashboards (`Voilà <https://github.com/voila-dashboards/voila>`__, `Voici <https://github.com/voila-dashboards/voici>`__). It’s based on the `Vuetify UI <https://v2.vuetifyjs.com/en/>`__ library. it has a lot more widgets than the standard jupyter widget library which, all of them being customizables and composables.

Reviewer: ```suggestion
**Ipyvuetify** is a widget library for making modern-looking GUIs in Jupyter notebooks (`classic <https://github.com/jupyter/notebook>`__, `lab <https://github.com/jupyterlab/jupyterlab>`__, `lite <https://github.com/jupyterlite/jupyterlite>`__) and dashboards (`Voilà <https://github.com/voila-dashboards/voila>`__, `Voici <https://github.com/voila-dashboards/voici>`__). Based on the `Vuetify UI <https://v2.vuetifyjs.com/en/>`__ library, it extends the standard Jupyter widget library with additional widgets that are more customizable and composable.
```

Reviewer: This fixes some spelling errors (also from parts that were in introduction before) and improved a bit by chatgpt.

Author: perfect, I merged them in.
